source/views/news/news-list.js

```diff
@@ -30,8 +31,10 @@ export class NewsList extends React.Component {
     })
   }
 
+  renderSeparator = () => <ListSeparator spacing={{left: 101}} />
+
   renderItem = ({item}: {item: StoryType}) =>
-    <NewsRow onPress={this.onPressNews} story={item} />
+    <NewsRow onPress={this.onPressNews} story={item} blank={this.props.thumb} />
```

Comment:
Why is this new prop called `blank`? Please change it to `thumb` or `thumbnail`.